Jenkinsfile

```diff
@@ -36,6 +36,8 @@ podTemplate(
         envVar(key: 'COGNITE_BASE_URL', value: "https://greenfield.cognitedata.com"),
         envVar(key: 'COGNITE_CLIENT_NAME', value: "powerdummy-populator")
     ]) {
+
+    deploySpinnakerPipelineConfigs {}
```

Comment:
Move this up to the second line of this file. It creates a separate pod when running the jenkins job, so it doesn't really belong inside this podTemplate.

Reply:
fixed